### PR TITLE
frontend/enhancement : added condition to check for hyderabad airport ride

### PR DESCRIPTION
**Pull request by Radhika1708**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Added if condition to check for airport ride for Hyderabad and play audio accordingly

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

**Comment by witcher-shailesh** [comment]
is it tested properly?

**Comment by Radhika1708** [comment]
> is it tested properly?

Sir, yesterday I tested in local, it worked fine. I also showed it to Praveen bhaiya. I booked a ride for Hyderabad airport, so after driver confirmed the ride, audio was playing. So, I am assuming it is working fine. If there are issues, kindly let me know.

File: Frontend/ui-driver/src/Screens/HomeScreen/View.purs [hunk 1]
```diff
@@ -2342,25 +2343,53 @@ getDriverStatusResult index driverStatus currentStatus= case (getValueToLocalSto
...
+    isAirportPriority = DS.contains(DS.Pattern("SureAirport")) (fromMaybe "" locationTag)
+    cityConfig = HU.getCityConfig state.data.config.cityConfig (getValueToLocalStore DRIVER_LOCATION)
+    vehicleAudioConfig = geVehicleBasedAudioConfig cityConfig.rideStartAudio vehicleType acRide
+
+    audioUrl = if isAirportPriority && isJust vehicleAudioConfig.parkingAudio then
+                 vehicleAudioConfig.parkingAudio
+               else if tollCharges > 0.0 && isJust vehicleAudioConfig.tollAudio then
+                 vehicleAudioConfig.tollAudio
+               else if isJust vehicleAudioConfig.acAudio then
+                 spy "vehicleAudioConfig.acAudio" vehicleAudioConfig.acAudio
```
Comment (spravinkumar9952) [inline Frontend/ui-driver/src/Screens/HomeScreen/View.purs:2372]: We can remove the spy

Comment (spravinkumar9952) [reply Frontend/ui-driver/src/Screens/HomeScreen/View.purs:2372]: return directly instead of storing in audio URL and return it

**Review by spravinkumar9952 — CHANGES_REQUESTED** [review]

File: Frontend/ui-driver/src/Screens/HomeScreen/View.purs (hunk 1 above)
Comment (Radhika1708) [reply Frontend/ui-driver/src/Screens/HomeScreen/View.purs]: Done

**Review by spravinkumar9952 — APPROVED** [review]

**Review by witcher-shailesh — APPROVED** [review]

**Pull request merged**